js/field.js

```diff
@@ -0,0 +1,228 @@
+function setOnClicksAndOtherThings() {
+    
+}
+
+var field = document.getElementById('field');
+var parentCard;
+var tempCard;
+var totalCountInAcePiles=0;
+field.onmousedown = onMouseDown;
+
+function onMouseDown(e) {
+    function isDragable(element) {
+        if (element.classList.contains('dragable')) {
+            return true;
+        }
+        return false;
+    }
+
+    function getCoords(element) {
+        var box = element.getBoundingClientRect();
+        return {
+            top: box.top + pageYOffset,
+            left: box.left + pageXOffset
+        };
+    }
+
+    if (isDragable(e.target)) {
+        var card = e.target;
+        card.style.zIndex = 90;
+        var top = card.style.top;
+        var left = card.style.left;
+        parentCard = card.parentNode;
+        var coords = getCoords(card);
+        var shiftX = e.pageX - coords.left;
+        var shiftY = e.pageY - coords.top;
+        card.style.position = 'absolute';
+        moveAt(e);
+
+        function moveAt(e) {
+            card.style.left = e.pageX - shiftX + 'px';
+            card.style.top = e.pageY -27 - shiftY  + 'px';//27 - это top-margin.
```

Comment:
Oh! Please don't use simple numbers inside your functions even with comments. Use constants instead. When you will need to change that value you'll probably loose it here...